biome.json

```diff
@@ -29,6 +29,12 @@
     "enabled": true
   },
   "linter": {
-    "enabled": false
+    "enabled": true,
+    "rules": {
+      "suspicious": {
+        "noConsoleLog": "off",
```

Comment:
Can we still get a `warn`?

Reply:
You would get a bunch of warnings, which are annoying I felt annoying. Ideally, we could use an override, but they are currently buggy. Are you still happy with a warning?

Reply:
Hm that is a tricky one. In general we don't really want to use `console.log()` for anything user-facing, everything user-facing should probably use astro's logger or error. So getting a warning, sounds feasible. However I was also annoyed by it before, so disabling this makes also sense.

I'm happy both ways, so maybe keep it off for now!